plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts

```diff
@@ -63,6 +65,18 @@ export function createFetchCookiecutterAction(options: {
             description: 'Values to pass on to cookiecutter for templating',
             type: 'object',
           },
+          copyWithoutRender: {
+            title: 'Copy Without Render',
+            description:
+              'Avoid rendering directories and files in the template',
+            type: 'object',
+          },
+          extensions: {
```

Comment:
What's your thought on how to use extensions in practice? I think this needs some additions to the documentation to explain how to actually get these extensions working in production. I'm assuming that you need to create your own custom cookiecutter container that installs the desired extensions to make them available to your organization?

Reply:
We can give a similar heads-up as in the cookiecutter docs:

> you need to make sure you have all the extensions installed, before running Cookiecutter on a template that requires custom Jinja2 extensions.

Reply:
Huh, where? Either way I'd say our case is a bit more complex as it needs to be added to the container or host installation depending on how you're running cookiecutter.

Reply:
Apologies, that comment wasn't clear. I was just quoting from their docs.

I have updated the input's description (https://github.com/backstage/backstage/pull/4956/files#diff-7f7d42d576e48c76bf3709e8219e6041341e2721c0faa508fb16880c42dcf080R77), which shows up at `/create/actions`. Should I mention it in any other place?

Reply:
I think we probably want to have working documentation about how to modify your dockerfile to install extensions that are picked up by the existing docker container at the very least. It's just going to raise questions on how to use this otherwise with no clear explanation. Can you work on this too @vinayvinay?

Reply:
Sure @benjdlambert. What's the correct place to document this bit? In the official docs, we redirect readers from Software Templates > Builtin actions => to `/create/actions`. So what you mention can be added to the `description` for `extensions`. Would you agree?

Reply:
Morning @benjdlambert, the code is fixed, and I've tested this works end-to-end (catalog-info.yaml => generated repo). I'd like to add working documentation, but where would you suggest I do that:
- on `/create/actions` page, or
- scaffolder-backend plugin's README, or
- in the official docs sitting next to, say, [Writing Custom Actions](https://backstage.io/docs/features/software-templates/writing-custom-actions)?

Reply:
I think a good start would be to create a folder in contrib and put it there with a sample dockerfile that can be used adding some extensions and how it works there for now.

Reply:
Here are the docs, @benjdlambert: [contrib/docker/cookiecutter-with-jinja2-extensions](https://github.com/vinayvinay/backstage/tree/vinay-fetch-cookiecutter-more-inputs/contrib/docker/cookiecutter-with-jinja2-extensions). Do they look good?

Also, while working on that doc, I realised that if one creates a custom Cookiecutter Docker image, they won't be able to use it without customising the `scaffolder-backend` plugin, because the `imageName` that's pulled isn't configurable, it is always `spotify/backstage-cookiecutter`. If that sounds like another `input` to `fetch:cookiecutter`, `input.imageName`, I could do a quick PR and make that configurable. WDYR?

Reply:
Actually, that was quick, here's what a follow-up PR will look like for customising the `imageName`: https://github.com/backstage/backstage/commit/f7b7ed998821d395e1d817a47d406882486fce3c.

I'll make it presentable if I get your 👍🏼 for the idea.

Reply:
Morning folks, a gentle reminder about this PR, please.

I've responded to your feedback, tested these configs, added docs in `contrib` and made a suggestion above to make the cookiecutter `imageName` customisable, to make it easy for users to add a custom cookiecutter image with extensions.

Reply:
You can probably cherry-pick that commit onto this PR too I think, so we can see everything together to see if it all makes sense :)

Reply:
OK @benjdlambert, I've cherry-picked it here.

Reply:
Hi @benjdlambert, could I ask you to take a look at this PR? We'd like to start using the new inputs soon, and would be good to do it without local customisations.